setup.sql

```diff
@@ -35,6 +35,7 @@ CREATE TABLE transactions (
     merchant_name VARCHAR(255),
     account_id INT NOT NULL,
     categoryID INT,
+    catergory VARCHAR(20),
     transactionDate DATE,
     merchant_name VARCHAR(255),
     amount DECIMAL(15, 2) NOT NULL,
```

Comment:
Still has the merchant_name twice

Reply:
Also 'category' is misspelled as 'catergory'